.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,26 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the
+// README at: https://github.com/devcontainers/templates/tree/main/src/ubuntu
+{
+  "name": "homebrew-core",
+  // Or use a Dockerfile or Docker Compose file. More info: https://containers.dev/guide/dockerfile
+  "image": "mcr.microsoft.com/devcontainers/base:jammy",
+  "features": {
+    "ghcr.io/meaningful-ooo/devcontainer-features/homebrew:2": {}
```

Comment:
This is untrusted.

Reply:
Ok, so I will first make a **PR** with my **vscode** settings and extensions in [Homebrew/brew](https://github.com/Homebrew/brew), and when the **PR** is accepted - then we can use it in **this project** as well